data_structures/src/serialization_helpers.rs

```diff
@@ -201,3 +202,150 @@ impl From<GenesisBlock> for GenesisBlockInfo {
         }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+enum UtxoSelectionStrategyName {
+    Random,
+    BigFirst,
+    SmallFirst,
+}
+
+impl From<UtxoSelectionStrategyName> for UtxoSelectionStrategy {
+    fn from(x: UtxoSelectionStrategyName) -> UtxoSelectionStrategy {
+        match x {
+            UtxoSelectionStrategyName::Random => UtxoSelectionStrategy::Random { from: None },
+            UtxoSelectionStrategyName::BigFirst => UtxoSelectionStrategy::BigFirst { from: None },
+            UtxoSelectionStrategyName::SmallFirst => {
+                UtxoSelectionStrategy::SmallFirst { from: None }
+            }
+        }
+    }
+}
+
+impl<'a> From<&'a UtxoSelectionStrategy> for UtxoSelectionStrategyName {
+    fn from(x: &'a UtxoSelectionStrategy) -> UtxoSelectionStrategyName {
+        match x {
+            UtxoSelectionStrategy::Random { .. } => UtxoSelectionStrategyName::Random,
+            UtxoSelectionStrategy::BigFirst { .. } => UtxoSelectionStrategyName::BigFirst,
+            UtxoSelectionStrategy::SmallFirst { .. } => UtxoSelectionStrategyName::SmallFirst,
+        }
+    }
+}
+
+#[derive(Deserialize, Serialize)]
+#[serde(untagged)]
+enum UtxoSelectionStrategyHelper {
```

Comment:
When I tried to use the Wallet API with a random strategy didn't work.

Input
```
    "utxo_strategy": {
      "strategy": "Random",
      "from": "wit1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqwrt3a4"
    }
```

Output:
```
    "message": "Invalid params: data did not match any variant of untagged enum UtxoSelectionStrategyHelper.",
```

Reply:
This is strange because the tests pass, can you provide the full JSON-RPC call?

Reply:
```json
{
  "jsonrpc": "2.0",
  "method": "create_vtt",
  "params": {
    "session_id": "663cffd566a614c6ac478d550a05e5c66c366b3ffab40b288d4812ce1a56e6b3",
    "wallet_id": "8f5b85981addad621a86f01a1ddb646ccd90620c95247948ce8d99feefd0496c",
    "fee": 1,
    "label": "",
    "outputs": [
      {
        "address": "twit1nfkythqds4r2hz3le2zaauxtl7yum76jd0ut9c",
        "amount": 1,
        "time_lock": 0
      }
    ],
    "utxo_strategy": {
      "strategy": "Random",
      "from": "wit1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqwrt3a4"
    }
  },
  "id": 18
}
```

Reply:
Another weird thing is that if I try to create a VTT with the strategy with valid UTXOs there is also an error.

Input:
```json
{
  "jsonrpc": "2.0",
  "method": "create_vtt",
  "params": {
    "session_id": "241ec1bbbadbd7024bccde1746f64124a0db6d5126800ea408aed21def0f7391",
    "wallet_id": "8f5b85981addad621a86f01a1ddb646ccd90620c95247948ce8d99feefd0496c",
    "fee": 1,
    "label": "",
    "outputs": [
      {
        "address": "twit1eghyyar76nuvdfu0h70f4gmxruj2rw4gfnrhnk",
        "amount": 1,
        "time_lock": 0
      }
    ],
    "utxo_strategy": {
      "strategy": "Random",
      "from": "twit1eghyyar76nuvdfu0h70f4gmxruj2rw4gfnrhnk"
    }
  },
  "id": 18
}
```

Output:
```json
{
  "jsonrpc": "2.0",
  "error": {
    "code": 400,
    "message": "Validation Error",
    "data": [
      [
        "balance",
        "Wallet account has not enough balance"
      ]
    ]
  },
  "id": 18
}
```

If I take away the `utxo_strategy` it works and creates the transaction.

Reply:
The Invalid params error is probably because the address is invalid, and it shows a generic error instead of "invalid address". That can be fixed.

Reply:
The second issue is strange because the node API works as expected, so it must be a problem in the wallet component

Reply:
We couldn't reproduce the "Wallet account has not enough balance" error, so assuming that it was because of external factors.